python/python/tests/test_async_client.py

```diff
@@ -926,14 +926,15 @@ async def test_unlink(self, redis_client: TRedisClient):
 
     @pytest.mark.parametrize("cluster_mode", [True, False])
     @pytest.mark.parametrize("protocol", [ProtocolVersion.RESP2, ProtocolVersion.RESP3])
-    async def test_expire_pexpire_ttl_with_positive_timeout(
+    async def test_expire_pexpire_ttl_pttl_with_positive_timeout(
```

Comment:
please tell me if you prefer:
```
@pytest.mark.parametrize("cluster_mode", [True, False])
    @pytest.mark.parametrize("protocol", [ProtocolVersion.RESP2, ProtocolVersion.RESP3])
    async def test_pttl(self, redis_client: TRedisClient):
        key = get_random_string(10)
        assert await redis_client.pttl(key) == -2
        current_time = int(time.time())

        assert await redis_client.set(key, "value") == OK
        assert await redis_client.pttl(key) == -1

        assert await redis_client.expire(key, 10) == True
        assert 0 < await redis_client.pttl(key) <= 10000

        assert await redis_client.expireat(key, current_time + 20) == True
        assert 0 < await redis_client.pttl(key) <= 20000

        assert await redis_client.pexpireat(key, current_time * 1000 + 30000) == True
        assert 0 < await redis_client.pttl(key) <= 30000

        # set command clears the timeout.
        assert await redis_client.set(key, "value") == OK
        assert await redis_client.pttl(key) == -1
```

Reply:
Yes, separating the command to a different test will help debugging if needed later on. The last check though seems duplicated (set and then asserting again that its PTTL is -1)